### DISCUSSION TITLE
Pi 4b / Bullseye / HQ camera working

### DISCUSSION THREAD

**Discussion by joetech** [photobooth-app/photobooth-app#153] [Howtos and camera specific topics]
Thanks for the software and good instructions.  It took almost no time at all to install and set up for my Pi HQ camera on Bullseye.  I'm building up an Allsky setup and realized quickly that I really wanted a way to see a live preview while I fidget with the focus and other controls.  Because of how I'm using it, I installed Bullseye Lite and run it headless, so the browser-based preview is ideal.

**Comment by mgineer85** [comment]
Nice to hear it works well for you and installation instructions are well made 🥰

Let me know where there is room for improvement, or feel free to edit and send pull requests.

Would you mind posting pictures of your setup here?
Give the repo a star?
What additional parameter you tweak? Maybe it could be integrated?


Have fun

**Comment by joetech** [comment]
Once I get to tweaking some, I'll share more.  I'll definitely send a PR if I have anything useful to add to the repo as I go.
